src/app/panels/righthand-panel.js

```diff
@@ -141,10 +141,19 @@ const css = csjs`
   }
   .dragbar             {
     position           : absolute;
-    width              : 0.5em;
+    width              : ${
```

Comment:
if you really need to have a specific value for this, that's fine no problem. 
But I would push back the original value (`0.5em` in that case) in the 2 others style.js (dark and light) instead of having a `if` here.